.devcontainer/devcontainer.env

```
@@ -1,5 +1,6 @@
 # Database environment vars
-MARIADB_ROOT_PASSWORD=rootpassword
+MARIADB_SERVICE_USER=naad_user
+MARIADB_SERVICE_PASSWORD=password
```

```suggestion
MARIADB_SERVICE_USER=root
MARIADB_SERVICE_PASSWORD=rootpassword
```
I think for now it'll have to run everything under `root` in the devcontainer because there's nothing that creates the user when the devcontainer runs. We can add this in a future ticket.

---

Yeah, I think we can run a command to add the user in the devcontainer json, but I was having trouble getting it working. Something like:
`"postCreateCommand": [the same command from the job]`

---

Yeah I played with it a bit to see if I could get that working but I couldn't. I think it's ok for now.